src/vcpkg/install.cpp

```diff
@@ -1003,6 +1019,50 @@ namespace vcpkg::Install
             {
                 extended_overlay_ports.push_back(paths.builtin_ports_directory().native());
             }
+
+            std::string hash;
+            {
+                auto timer = ElapsedTimer::create_started();
+                auto hasher = Hash::get_hasher_for(Hash::Algorithm::Sha256);
+                for (auto&& path : args.overlay_ports)
+                {
+                    for (auto&& file : fs.get_regular_files_recursive(path, VCPKG_LINE_INFO))
+                    {
+                        hasher->add_bytes(fs.read_contents(file, VCPKG_LINE_INFO));
```

Comment:
cc @ras0219 don't we do something interesting for the hashing of port contents?

Reply:
Yes, this hashing method doesn't include file names so renaming a file in an overlay will not result in a rebuild.